src/theme/Colors.ts

```diff
@@ -6,4 +6,5 @@ export const Colors = {
     RedBaron : '#BC0016',
     lightRed : '#E22540',
     aubergine : '#201D1E',
+    Nero  : '#181818',
```

Comment:
@Nitish-Kes Please try to use names relevant to usage and not their actual names. This will help other developers identify colors easily in code.
e.g. lightgray, textInputColor, defaultRed etc.